Geometry/CMSCommonData/python/cmsExtendedGeometry2021XML_cfi.py

```diff
@@ -346,6 +344,8 @@
         'Geometry/VeryForwardData/data/CTPPS_Diamond_2021/CTPPS_Diamond_Planes/CTPPS_Diamond_Plane3.xml',
         'Geometry/VeryForwardData/data/CTPPS_Diamond_2021/CTPPS_Diamond_Planes/CTPPS_Diamond_Plane4.xml',
         'Geometry/VeryForwardData/data/CTPPS_Diamond_2021/CTPPS_Diamond_Detector_Assembly.xml',
+        'Geometry/VeryForwardData/data/CTPPS_Diamond_2021/CTPPS_Detectors_Assembly_022.xml',
+        'Geometry/VeryForwardData/data/CTPPS_Diamond_2021/CTPPS_Detectors_Assembly_122.xml',
```

Comment:
Are these: Geometry/VeryForwardData/data/RP_Detectors_Assembly/2021/Simu/CTPPS_Detectors_Assembly_022.xml?
If so, I would use Geometry/VeryForwardData/data/CTPPS_Diamond_2021/CTPPS_Detectors_Assembly_022.xml and remove Geometry/VeryForwardData/data/RP_Detectors_Assembly/2021/Simu/CTPPS_Detectors_Assembly_022.xml.
same for 122.

Reply:
Strange, I don't see these files in my local repo:

Geometry/VeryForwardData/data/CTPPS_Diamond_2021/CTPPS_Detectors_Assembly_022.xml
Geometry/VeryForwardData/data/CTPPS_Diamond_2021/CTPPS_Detectors_Assembly_122.xml

since I dropped them to keep them at RP_Detectors_Assembly/2021/Simu/ as you said. I believe they'll be dropped in the other commit, no?

Reply:
Indeed, the last commit will drop these files. I'm sorry for the messy PR, I thought git would organize it better.

Reply:
Sorry. Let's try to restart from scratch.
CTPPS_Detector_Assembly_*22.xml are used only for Simu in 2021. I see now that putting them inside RP_Detectors_Assembly violates the rule of naming schemes. Why don't we put them inside CTPPS_Diamond_2021/Simu/v1 and just comment RP_Detecor_Assembly_*22.xml in dict2021?

Reply:
ok sounds good. I'll leave the change in the dict2021 for you.

Reply:
fine

Reply:
I'll close this PR and make a clear one.